pkg/plugins/api.go

```diff
@@ -0,0 +1,48 @@
+package plugins
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConsolePlugin is an extension for customizing OpenShift web console by
+// dynamically loading code from another service running on the cluster.
+type ConsolePlugin struct {
```

Comment:
These declarations seem to be copied from openshift/api#764 - isn't it better to reference them instead? (as in go package reference or similar)

Reply:
It is, and will remove it. But it's faster 😄

Reply:
Also there is an issue when trying to replace the openshift/api with my jhadvig/api branch that contains the API changes, which was introduced by https://github.com/openshift/console/commit/bdf02e6b5157274ae9debda32f6800354e47bc8a